dist/js/iosCheckbox.js

```diff
@@ -57,9 +71,6 @@
                         // Update state
                         org_checkbox.removeAttr('checked');
                     }
-                    
-                    // Run click even in case it was registered to the original checkbox element.
-                	org_checkbox.click();
```

Comment:
Removed code snippet as it was leaving the button active and the checkbox field inactive (unchecked)

Reply:
👍